Add support for latest release (#323)
erlef/setup-beam: README.md

specifier (`v1.11.0-rc.0`) and set option `version-type` to `strict`. Pre-release versions are
opt-in, so `1.11.x` will not match a pre-release.

#### "Stable" versions

Set a tool's version to `stable` to retrieve the latest non-RC version of a given tool.
The latest version is (locally) calculated by the action based on the (retrieved) versions
it knows (**note**: it is not the same as [GitHub considers it](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
and some repositories might propose).

If in doubt do a test run and compare the obtained release with the one you were expecting to
be the latest.

#### "Edge" versions

Similar to `stable`, except that release candidates are also considered.

> Currently, the same behaviour can be achieved with the deprecated `latest` alias.

Reviewer: ```suggestion
To target these, set a tool's version to `edge` (same behaviour as deprecated `latest`).
```

Reviewer: ("Currently" will eventually stop making sense)

Author: Definitely. If you have a preferred README text i will adjust it to that.

Reviewer: The one I added above, for example. You can commit my suggestion or do it locally, and push, whatever's better for you.

specifier (`v1.11.0-rc.0`) and set option `version-type` to `strict`. Pre-release versions are
opt-in, so `1.11.x` will not match a pre-release.

#### "Latest" versions
#### "latest" versions

Set a tool's version to `latest` to retrieve the latest version of a given tool.
Set a tool's version to `latest` to retrieve the latest non-RC version of a given tool.
The latest version is (locally) calculated by the action based on the (retrieved) versions
it knows (**note**: it is not the same as [GitHub considers it](https://docs.github.com/en/repositories/releasing-projects-on-github/managing-releases-in-a-repository)
and some repositories might propose).

If in doubt do a test run and compare the obtained release with the one you were expecting to
be the latest.

#### "latest-rc" versions

Similar to `latest`, except that release candidates are also considered.
To target these, set a tool's version to `latest-rc`.

### Compatibility between Operating System and Erlang/OTP

This list presents the known working version combos between the target operating system
and Erlang/OTP.

| Operating system | Erlang/OTP   | OTP Architecture | Status

> |-                 |-             | -                |-

Reviewer: I think the only broken thing was the space after `|` below "OTP Architecture". I'd like to keep as is, instead of aligning the code visually since, e.g. a future change might imply changing multiple lines, which won't be good.

Self-hosted runners need to set env. variable `ImageOS` to one of the following, since the action
uses that to download assets:

> | ImageOS    | Operating system

Reviewer: I don't think this was broken. Why was it updated?
